google/resource_compute_firewall.go

```diff
@@ -302,11 +457,12 @@ func resourceFirewall(
 	}
 
 	// Build the firewall parameter
-	return &compute.Firewall{
+	return &computeBeta.Firewall{
 		Name:         d.Get("name").(string),
 		Description:  d.Get("description").(string),
 		Network:      network.SelfLink,
```

Comment:
This variable never gets stored in state, since we do that in `Read`.

We can just read at `v1`; done.